microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+
+In the following image, the **Focus Mode** UI is shown.  **Focus Mode** has a **Quick View** area at the bottom, in place of the **Drawer**.  The **Layers** tool, which is shown open in **Quick View**, contains only a link, to open the **3D View** tool above it:
+
+![The Layers tool now links to the 3D View tool with the Composited Layers view open.](../../media/2022/03/layers-3d-view-tool.png)
+<!-- work item > layersmove.gif -->
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
+
+
+<!-- ====================================================================== -->
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
+
+Starting in Microsoft Edge version 99, you can unminify a call stack in a performance profile, as follows:
+1. After recording a profile, in the **Performance** tool, click the new **Unminify** button.
+1. Download the unminified performance profile.<!--how?-->
+1. Import the unminified performance profile into the **Performance** tool.<!--how?-->
+
+The **Unminify** button will use source maps, provided that they are hosted side-by-side with your production code, to unminify the call stack in the **Performance** tool.
+
+The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, and **bn**:
+
+![The minified call stack in the Performance tool.](../../media/2022/03/minified-call-stack-performance-tool.png)
+<!-- Image 1: Minified call stack in Performance tool
+Navigate to: https://outlook-sdf.office.com/mail/
+Open DevTools > Performance
+Record a performance profile while scrolling through your inbox
+Select a small section of the profile and look at the call stack       1380ms-1400ms
+Draw red highlights around function names like n, O, i, Xt, Bn, Yn. It won't be these exactly in your screenshot but you can see these are minified function names
+Remember the section of the profile that you are looking at
+-->
+
+The unminified call stack in the **Performance** tool now shows those functions labeled with meaningful names: **invokeFunc (b)**, **executeAction (O)**, **endBatch (Xt)**, and **runReactions (bn)**:
+
+![The unminified call stack in the Performance tool.](../../media/2022/03/unminified-call-stack-performance-tool.png)
```

Comment:
Please crop the title bar out of this screenshot as well

Reply:
Erased URL.  Is that sufficient, or must the titlebar be removed entirely?  
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1779#use-your-source-maps-to-unminify-performance-profiles

Reply:
Works for me. Thanks Michael!